osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/application/config/AbstractSchedulingConfig.java

```diff
@@ -40,24 +65,25 @@ public SpringBeanJobFactory springBeanJobFactory() {
     }
 
     /**
-     * Construct the scheduler taskpool with specified job and trigger
+     * Construct the scheduler task-pool with specified job and trigger.
      *
      * @param schedulingConfigProperties
-     *            an object containing all the properties needed to configure
-     *            the Quartz scheduler instance
-     * @return the Quartz scheduler instance
+     *            Object containing all the properties needed to configure the
+     *            Quartz scheduler instance.
+     *
+     * @return The Quartz scheduler instance.
+     *
      * @throws SchedulerException
-     *             when issues occur in constructing schedules
+     *             When issues occur in constructing schedules.
+     *
+     * @deprecated Use
+     *             {@link AbstractSchedulingConfig#constructAndStartQuartzScheduler(SchedulingConfigProperties)}.
      */
+    @Deprecated
     protected Scheduler constructScheduler(final SchedulingConfigProperties schedulingConfigProperties)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1133)